questionnaires/migrations/0003_auto_20210615_0552.py

```diff
@@ -0,0 +1,92 @@
+# Generated by Django 3.1.12 on 2021-06-15 05:52
```

Comment:
there are 3 migration files. its good to make only one migration per module per PR.

you can revert migrations, delete these files and re-create migration - so there will be only one migration file